hydra-node/test/Hydra/Chain/Direct/Contract/Close.hs

```diff
@@ -209,6 +211,7 @@ genCloseMutation (tx, _utxo) =
                 (replacePolicyIdWith Fixture.testPolicyId otherHeadId healthyOpenHeadTxOut)
                 (Just $ toScriptData healthyCloseDatum)
             ]
+    , SomeMutation (Just "wrong datum hash") MutateContesters . ChangeOutput 0 <$> mutateClosedContesters
```

Comment:
Should: not check this via comparing hashes, but in the style of #677 already. Likely we will merge this after that other branch. So we expect an error like "non empty contesters" or so.

Reply:
the error being thrown in this case is "changed parameters" to match the mentioned style.

a condition `&& null contesters'` has been added to `mustNotChangeParameters` in `checkClose`

Reply:
We separated the check in a different step.